codecov.yml

```diff
@@ -51,3 +51,7 @@ flag_management:
       paths:
         - ".*"
       carryforward: true
+
+ignore:
+  - "homeassistant/scripts/benchmark"
+  - "homeassistant/scripts/macos"
```

Comment:
We have stopped ignoring coverage, so I'm not sure why we re-introduce it here?

../Frenck